### PR TITLE
Type coercion

### PR DESCRIPTION
**Pull request by JS-goose**
I'm not 100% exactly sure what I am doing is correct so if this isn't right feel free to disregard this pull request :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,47 +1,56 @@
...
+					<a class="nav-link" href="#">Topic C</a>
+					<a class="nav-link" href="#">Topic D</a>
+				</nav>
+			</div>
+			<div class="col-sm-9 doc-main">
+				<div class="card">
+					<div class="card-header">Welcome</div>
+					<div class="card-body">
+						<p>This site revolves around the learner. We wanted to give people a documentation page that taught concepts with a less
+							technical approach. When someone first jumps into programming, documentation can be an intimidating body of writing.
```
Comment (jeneesmith1) [inline index.html]: Hello - I am your friendly content reviewer! I think this should be revised to   --- "We wanted to give people a documentation page that teaches concepts with a less technical approach."

Original

> We wanted to give people a documentation page that taught concepts with a less
technical approach.

Comment (jeneesmith1) [inline index.html]: I think this should be revised to - -- "When someone first jumps into programming, documentation can be intimidating to read and understand." 

Original 

> When someone first jumps into programming, documentation can be an intimidating body of writing.

File: index.html [hunk 2]
```diff
@@ -1,47 +1,56 @@
...
+							And yet, at times it is their only source of help. So, in order to assist the beginnger, we've endeavored to provide
```
Comment (jeneesmith1) [inline index.html]: I think this should be revised to - "And yet, at times it is the learner's only source of help."

Original 

> And yet, at times it is their only source of help.

File: index.html [hunk 3]
```diff
@@ -1,47 +1,56 @@
...
+							help by breaking down the concepts into simpler terms with illustrative explanations. We hope you find the documentation
+							contained within this site useful for your education of all things Javascript.</p>
```
Comment (jeneesmith1) [inline index.html]: Revise to -- "We hope you find the documentation in this site useful for your education on all things Javascript."

Original - 

> We hope you find the documentation
contained within this site useful for your education of all things Javascript.

File: index.html [hunk 4]
```diff
@@ -1,47 +1,56 @@
...
-					
 				</div>
-				<div class="contribute col-sm-12">
-				<p > Want to contribute? Go <a href="https://github.com/dalumiller/Udacity-Documentation-Group-Project">here</a> to get started.</p>
+				<div class="card">
+					<div class="card-header">The Vision</div>
+					<div class="card-body">
+						<p>As stated, this site revolves around the learner. The pages you'll see were written by individuals new to programming.
+							Most have the Grow with Google Scholarship: Front End Web Dev program through Udacity to thank for their burgeoning
+							coding knowledge. For those individuals, this site was invented to give them additional coding experience, an opportunity
```
Comment (jeneesmith1) [inline index.html]: Revise this to -- "As stated, this site revolves around learners. The pages here were written by individuals new to programming. Most have taken the Grow with Google Scholarship: Front End Web Dev program through Udacity as an introduction to our burgeoning knowledge."

Original 
> "As stated, this site revolves around the learner. The pages you'll see were written by individuals new to programming. Most have the Grow with Google Scholarship: Front End Web Dev program through Udacity to thank for their burgeoning coding knowledge."

File: index.html [hunk 5]
```diff
@@ -1,47 +1,56 @@
...
+							contained within this site useful for your education of all things Javascript.</p>
 					</div>
-					
 				</div>
-				<div class="contribute col-sm-12">
-				<p > Want to contribute? Go <a href="https://github.com/dalumiller/Udacity-Documentation-Group-Project">here</a> to get started.</p>
+				<div class="card">
+					<div class="card-header">The Vision</div>
+					<div class="card-body">
+						<p>As stated, this site revolves around the learner. The pages you'll see were written by individuals new to programming.
```
Comment (jeneesmith1) [inline index.html]: I think that you should consider using a `<strong>` or `<em>` tag to show our course name!

File: index.html (hunk 4 above)
Comment (jeneesmith1) [inline index.html]: Revise this to -- "This site was invented by a group from that course to give us additional coding experience, an opportunity to explain a Javascript concept, and an opportunity to collaborate."

original 
> "For those individuals, this site was invented to give them additional coding experience, an opportunity to explain a Javascript concept, and an avenue to participate in a collaborative endeavor."

File: template.html [hunk 6]
```diff
@@ -1,79 +1,91 @@
...
+	<link rel="stylesheet" type="text/css" href="css/stylesheet.css" />
+</head>
+
+<body>
+	<div class="doc-masthead">
+		<h1>Doc
+			<img id="logo" src="img/udacity-logo.jpg">mentation Project</h1>
+	</div>
+	<div class="doc-header">
+		<h2>Impliciti Type Coercion</h2>
```
Comment (jeneesmith1) [inline template.html]: Did you mean to write "Impliciti" here? I think you may have meant to write "Implicit", but I just wanted to check.

File: template.html [hunk 7]
```diff
@@ -1,79 +1,91 @@
...
-  						<div class="card-body">
-      					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer posuere erat a ante.</p>
-      					</div>
+				</div>
+				<div class="card">
+					<div class="card-header">Possible Uses</div>
+					<div class="card-body">
+						<p>For the vast majority of the time, you
+							<strong>will not</strong> want to use type coercion. Now, you may be asking yourself why we took the time to explain something
+							to you that you will, in all likelihood, not want to use regularly. The answer is simple -- you need the knowledge
```
Comment (jeneesmith1) [inline template.html]: Revise this to  -- 
"Now, you may be asking yourself why we took the time to explain something that you will, in all likelihood, not want to use regularly."

Original
> Now, you may be asking yourself why we took the time to explain something
to you that you will, in all likelihood, not want to use regularly.

File: template.html [hunk 8]
```diff
@@ -1,79 +1,91 @@
...
-					<div class="card">
-  						<div class="card-header">Examples</div>
-  						<div class="card-body">
-      					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-      					</div>
+				</div>
+				<div class="card">
+					<div class="card-header">The Call</div>
+					<div class="card-body">
+						<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer posuere erat a ante.</p>
```
Comment (jeneesmith1) [inline template.html]: Is there a reason you left the Lorem ipsum in the card body? It seems like you would want to remove or revise this.

**Comment by JS-goose** [comment]
Hey Jenee, my apologies about all the conflicts.  This is my first pull request so I'm not exactly sure how to fix these.  Should I just fix these issues locally then push to my forked repo and then make another pull request?  Also, should I complete my portion completely before making a pull request?  I'm sorry about the questions, I just want to make sure I do this right.  Thank you in advance for all your help!

**Comment by jeneesmith1** [comment]
Don't apologize! Editing your work to make it better is part of the process! 
Yes, you can make those changes on your local branch, and then push them to your forked repo. HINT: run ``git status`` to see what branch you are on, and run ``git diff`` from your terminal to see the branch you are on, as well as the changes you have made before you push your changes. The updates you make should be viewable without you needing to make a new pull request. And while you would want to have something close to completely done by the time you make a pull request, most people have to make edits during the review process. I think that the process makes the final product that much better! 👍

**Comment by JS-goose** [comment]
Thank you for explaining the process and answering my questions.  I feel much better about my pull request now :D  I will make the necessary changes and push them to my forked repo.  Again, thank you for taking the time to answer my questions and having patience with me :)

**Comment by jeneesmith1** [comment]
Hey @JS-goose this looks good! The last thing you will need to do is resolve your merge conflicts. This happens when more than one person makes a change in the same file. These are easy to fix. You can do that from the command line by typing `git pull origin master`, which will try to integrate your branch with the master branch. It will tell you that you have some conflicts that must be resolved before you can push your changes. Open index.html and template.html. You will want to look for the `<<<<<HEAD` of your conflict, as well as a commit number message at the bottom of the conflicting text.  Fix those, and then `git add` them to address the issue. You can then commit and push the changes to your branch, and that should remove the merge conflicts. 

Once that's done, you should be ready to merge your changes!

**Comment by jmaldia** [comment]
@JS-goose, you're supposed to save your work on a separate html file . The template shouldn't be touched.

http://screen.ac/441H2D3Q2W1o

**Comment by JS-goose** [comment]
Thank you for pointing that out @jmaldia, I made the necessary change :)

@jeneesmith1  I'll make the necessary changes ASAP and thank you again for all the help!  I really appreciate you taking the time to explain all of this to me 👍 👏

**Comment by dalumiller** [comment]
typecoercion
Hey Jon, when I opened up your branch this is what I found for your typeCoercion page. Also, it appears that you edited the original template and index page, because when I tried to open them the Udacity logo was gone as well as the styling. Please re clone/fork my repo to start with a fresh set of those files and then add your typeCoercion page into it. Then call for a new pull request.

**Comment by jeneesmith1** [comment]
Hello @JS-goose @dalumiller I think you may have renamed your files, when instead what you might want to do is create a duplicate file that leaves the index and template files untouched! If you try to push this branch, you will be erasing the template files for everyone using the repository.

**Comment by JS-goose** [comment]
Ok, I will re clone/fork the repo to start fresh.  Sorry about that, first time actually doing a pull request on something that wasn't a tutorial lol

**Comment by JS-goose** [comment]
I believe I did this correctly this time but I wanted to check before I make a new pull request.  I created typeCoercion.html (with the same information I had before) inside the "topics" folder after I deleted my old fork and then reforked the original repo. I haven't touched the index.html or template.html

Would anyone be able to tell me if this is correct?  Thank you in advance!

**Comment by jmaldia** [comment]
@JS-goose It looks like you made the correct file for the topic. https://github.com/JS-goose/Udacity-Documentation-Group-Project/blob/typeCoercion/topics/typeCoercion-page.html

However, I don't see the styles. 

Also, it looks like your codebase maybe at least 1 merge behind. 

Maybe do a git pull first on your dev branch and then switch back to your feature branch and then merge. 

I suggest backing up any changes you have first just to make sure that you don't lose any of your work.

**Comment by JS-goose** [comment]
Maybe someone can help me with this...in my branch (typeCoercion) I had to change the relative path for the images and stylesheet from `href="css/stylesheet.css"` to `../css/stylesheet.css` in order for the styling to appear...however I created another branch named fix-path-jjgoose and the path is `href="css/stylesheet.css"` yet the styling loads....I am very confused lol

**Comment by jmaldia** [comment]
@JS-goose where does the styling load? As we talked about, the path needs to be changed when moved to the topics page. I think this is something that we should address. @dalumiller, we should change the path on template.html because when someone adds the file to the topics folder, the current file path would no longer be valid. You need to move up one folder. So, as Jonathan mentioned, you need to change href="css/stylesheet.css" to ../css/stylesheet.css (and all file paths).

**Comment by dalumiller** [comment]
This still has a conflict with index.html and template.html. Address those then re-issue on pull-request

**Pull request closed without merging**